composer.json

```diff
@@ -18,7 +18,7 @@
     "require-dev": {
         "symfony/config": "2.*",
         "phpunit/phpunit": "3.7.*",
```

Comment:
That's right, hehe, I wasn't aware that we currently don't do that, but we should :)